js/test/date/testcalendar.js

```diff
@@ -23,14 +23,14 @@ if (typeof(LocaleInfo) === "undefined") {
 
 module.exports.testcalendar = {
     testDefaultCalendar_GregorianCaseAll: function(test) {
-        test.expect(193);
+        test.expect(195);
         var localeList = ["ar-AE","ar-BH","ar-DJ","ar-DZ","ar-EG","ar-IQ","ar-JO","ar-KW","ar-LB","ar-LY","ar-MA","ar-MR",
-        "ar-OM","ar-QA","ar-SA","ar-SD","ar-SY","ar-TN","ar-YE","as-IN","az-Latn-AZ","bg-BG","bn-IN","bs-Latn-BA,bs-Latn-ME",
+        "ar-OM","ar-QA","ar-SA","ar-SD","ar-SY","ar-TN","ar-YE","as-IN","az-Latn-AZ","be-BY","bg-BG","bn-IN","bs-Latn-BA,bs-Latn-ME",
```

Comment:
Are all these locale test branches independent of each other? If so, be-BY should not appear in this list in this branch

Reply:
Actually, I wanted to separate PRs per local to see the change clearly. but it modifies the same files at the same time and can't expect when it's merged. Probably it's not a good idea to make a PR per locale.. I need to make test cases for 8 locales. probably making one PR would be better.